rpc/rpc-wrappers.go

```diff
@@ -670,6 +671,26 @@ func (pub PublisherClient) SubmitToCT(ctx context.Context, der []byte) (err erro
 	return
 }
 
```

Comment:
Addressed in https://github.com/letsencrypt/boulder/pull/2342/commits/0e8dc75207f31c1f4abbeedd02fcea6c67076f9d